platform/admin/core/admin-core-appservice/src/main/java/org/codice/ddf/admin/application/service/impl/ApplicationServiceBean.java

```diff
@@ -167,33 +175,57 @@ public void destroy() throws ApplicationServiceException {
     }
   }
 
+  @Override
+  public void installProfile(String profile, boolean noRefresh) {
+    try {
+      EnumSet<FeaturesService.Option> options = EnumSet.noneOf(FeaturesService.Option.class);
+      if (noRefresh) {
+        options.add(FeaturesService.Option.NoAutoRefreshBundles);
+      }
+      featuresService.installFeature(profile, options);
```

Comment:
Should the try/catch surround only this line, the reset don't seem to throw any exceptions?

Reply:
If any failures occur in these methods we will want to log for context since the third party libraries don't do a good job of it for us.